### ISSUE TITLE
[BUG] Conversion to 2.0.0 Trait & Constraint connection

### ISSUE THREAD

**Issue by catroest** [eclipse-esmf/esmf-aspect-model-editor#36]
**Describe the bug**
The bug appears in the editor when opening a model with version 1.0.0. Multiple constraints/traits are not correctly associated with the corresponding property. So no property links to (here in the example, these two) traits.

**Where**
The problem appears in the editor in the version: 
AME version:
4.2.1

BAMM version:
2.0.0

**Screenshots**
![image](https://user-images.githubusercontent.com/92307971/231466325-b70f0070-8aa2-4cba-aece-3249970c74c3.png)

**Additional context**
Some other traits where loaded correctly. It could be connected to situation that several properties point to these traits.

**Comment by chris-volk** [comment]
Dear @catroest , sorry for reacting slowly. Do you have a (preferably small) example ttl file illustrating this issue? Thanks very much.

**Comment by michelu89** [comment]
Dear @catroest,

after examining your example and reviewing the original file on https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.pcf, I observed that the two traits have missing properties.

```
:NonEmptyStringConstraint a samm-c:LengthConstraint ;
   samm:preferredName "Non Empty String Constraint"@en ;
   samm:description "Constraint for ensuring that a string has at least one character."@en ;
   samm-c:minValue "1"^^xsd:nonNegativeInteger .
 
:NonEmptyStringTrait a samm-c:Trait ;
   samm-c:baseCharacteristic :NonEmptyString ;
   samm-c:constraint :NonEmptyStringConstraint .
```

It is possible that something went wrong during the export in version v4.2.1, which resulted in the properties not being included.

Regrettably, I was unable to replicate this issue in both v4.2.1 and the latest release, v4.3.0.

I would recommend trying out the newest version and reinserting the properties.

Please be aware that v4.3.0 currently has a bug regarding the validation and export of units, but this will be addressed in the upcoming version.

To resolve this, you will need to manually modify the following for the unit after exporting it with v4.3.0. (Unfortunately, this can only be validated with the [samm-cli](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.2.1)

```
:Emission a samm:Unit ;
   samm:preferredName "Emission"@en ;
   samm:symbol "kgCO2e/kg" .
```

**Comment by catroest** [comment]
After a little deeper research, it was noticed that the original file also does not have the connection between Property and Trait. It looks like one of our modelers made a mistake here. This is the error and what it should look like:
![image](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/92307971/899329e4-0a2c-442f-8e08-15225fc84c7d)


But then the problem here is that this is not shown as an error. Unless there is a legitimate case that a trait may be described without being called by a property, I would suggest that this be flagged as an error by the cli.

I'm sorry if there was any major extra work here for this error on our end.

**Comment by catroest** [comment]
For the unit error:
After changing to samm and saving the change in the turtle file, it will be reverted again after opening the file with the editor.
Is this intended?

**Comment by catroest** [comment]
Thank you for the information!

**Comment by michelu89** [comment]
Should be fixed on [v4.4.0](https://github.com/eclipse-esmf/esmf-aspect-model-editor/releases/tag/v4.4.0)
